Makefile

```diff
@@ -14,17 +15,17 @@ SCRIPTS = \
 
 .PHONY: install
 install:
-	install -dm0755 $(DESTDIR)$(SHAREDIR)
+	install -dm0755 $(DESTDIR)$(BINDIR)
 	for script in $(SCRIPTS); do \
-		install -Dm755 $${script} -t $(DESTDIR)$(SHAREDIR)/contrib/$${script%/*}; \
+		install -Dm755 $${script} $(DESTDIR)$(BINDIR)/$${script#*/}; \
```

Comment:
Wouldn't `install -Dm 755 $${script} -t $(DESTDIR)$(BINDIR)/` work as well?

Reply:
Argh, it does! But I reckon the substring stuff has to be there for the uninstall part.

Reply:
Hm, it does work without the `-t`?
I think it would fail at the first item

Reply:
From `man install`:

```
-D     create all leading components of DEST except the last, or all components of --target-directory, then copy SOURCE to DEST
```

I *think* it would not create `bin` then... or? If we add `-t` we'll definitely create the target dir entirely.

Reply:
`install -dm0755 $(DESTDIR)$(BINDIR)` before the for-loop. So that should be covered.

Reply:
You could remove the install before the for loop then. :)